### PR TITLE
Klarna Conf reference - Update for 2.4

### PR DESCRIPTION
**Pull request by samoko85**
## Purpose of this pull request

Updates for 2.4 release

## Affected documentation pages

https://docs.magento.com/user-guide/configuration/sales/klarna.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/configuration/sales/klarna.md [hunk 1]
```diff
@@ -38,8 +38,21 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
-|Details color|Store View|The [hexadecimal code](https://www.w3schools.com/colors/colors_picker.asp) for the background color of details that appear on the Klarna page during checkout. For example: #0072CC|
-|Border color|Store View|The hexadecimal code for the color of borders on the Klarna page during checkout. For example: #0072CC|
-|Selected border color|Store View|The hexadecimal code for the color of borders around the active section on the Klarna page during checkout. For example: #0072CC|
-|Text color|Store View|The hexadecimal code for the color of the main text on the Klarna page during checkout. For example: #0072CC|
-|Border Radius|Store View|The number of pixels that define the size of the curve at each corner of a box with borders. For example: 46px|
+|Details color|Website|The [hexadecimal code](https://www.w3schools.com/colors/colors_picker.asp) for the background color of details that appear on the Klarna page during checkout. For example: #0072CC|
```
Comment (ccondra) [inline src/configuration/sales/klarna.md:41]: The "Store View" scope setting for the Design section is correct. It did not change to "Website".

File: src/configuration/sales/klarna.md [hunk 2]
```diff
@@ -24,12 +24,12 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
...
-|Enable B2B|Store View|Allows B2B payments, if previously set up in your Klarna account. Options: Yes / No|
-|Enable Data sharing|Store View|Determines if customer data is shared with Klarna before the customer places an order. Sharing data allows Klarna to personalize credit offerings for the customer. Options: Yes / No|
-|Enable Data sharing on load|Store View|Allows you to determine when customer data is shared with Klarna. Options:<br />**No** - Customer billing and shipping address information is shared with Klarna after the transaction is authorized. If a shipping address error is encountered during authorization, the customer is instructed to return to the previous step and correct the problem.<br />**Yes** - Customer billing and shipping address information is shared with Klarna when the payment method is selected during the checkout process.|
-|Sort Order|Store View|A number that determines the sequence in which Klarna appears with other payment methods during checkout. If left blank, Klarna appears next in the list of active payment methods.|
+|Enable|Website|Enables Klarna for the current Website. Options: Yes / No|
```
Comment (ccondra) [inline src/configuration/sales/klarna.md:27]: The "Store View" scope setting is correct for items in this section.

**Comment by ccondra** [comment]
This PR can be closed without further action.

**Comment by samoko85** [comment]
Closed as per above comment

**Pull request closed without merging**